### ISSUE TITLE
Spec

### ISSUE THREAD

**Issue by aneesh-neelam** [aneesh-neelam/VITacademics-Worker#3]
Let's spec out the system. We need the following: 
- GCM dispatch
- iOS dispatch
- CRON?

**Comment by kishore-narendran** [comment]
Have we considered using Urban Airship for iOS and Android dispatch (Apparently they do both now)? 
Check this - http://urbanairship.com/products/mobile-app-engagement#push-messages
And what about for Windows Phone?

**Comment by karthikb351** [comment]
No, no Urban Airship. We can do this ourselves, it's really simple.

**Comment by aneesh-neelam** [comment]
I'm not sure how it works on Windows. Let's worry about Windows Phone a little later.

The Windows app is being developed [here](https://github.com/VinayGupta23/VITacademics_Windows). 

@VinayGupta23 says the Phone app is almost ready for a public beta release.

**Comment by kishore-narendran** [comment]
iOS has a GCM equivalent that is simple to use is it?

**Comment by sreeram-boyapati** [comment]
Apple Push Notification Service? I think there is something like that.

**Comment by biocross** [comment]
Here: https://developer.apple.com/library/ios/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/Chapters/ApplePushService.html
